test/extended/security/supplemental_groups.go

```diff
@@ -28,19 +32,22 @@ const (
 var _ = g.Describe("[sig-node] supplemental groups", func() {
 	defer g.GinkgoRecover()
 
-	var (
-		oc = exutil.NewCLI("sup-groups")
-		f  = oc.KubeFramework()
-	)
+	oc := exutil.NewCLIWithPodSecurityLevel("sup-groups", admissionapi.LevelBaseline)
+	ctx := context.Background()
 
 	g.Describe("Ensure supplemental groups propagate to docker", func() {
-		g.It("should propagate requested groups to the container [Local][apigroup:user.openshift.io][apigroup:security.openshift.io]", func() {
+		g.It("should propagate requested groups to the container [apigroup:security.openshift.io]", func() {
 
 			fsGroup := int64(1111)
 			supGroup := int64(2222)
 
+			projectName := oc.Namespace()
+			sa := createServiceAccount(ctx, oc, projectName)
```

Comment:
reusing this func from scc.go, should I moveit to a separate file like `common.go`?

Reply:
I'm not sure whether it was me who originally reviewed the PR of that SA creation, but this wait piece of the code is weird:

https://github.com/openshift/origin/blob/79baa1b40aa634e7a29b5c01abf756d70c58c0a4/test/extended/security/scc.go#L479-L491

If you're to re-use it, replace it with
https://github.com/openshift/origin/blob/79baa1b40aa634e7a29b5c01abf756d70c58c0a4/test/extended/util/framework.go#L1023-L1024

No need to export it or move it anywhere yet, I think.

Reply:
+1, using the `WaitForServiceAccount` now